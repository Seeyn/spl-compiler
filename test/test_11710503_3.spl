struct foo{
    int a[10];
};
int main(){
    int i = nononono();
    int j = 1;
    struct foo k;
    k.a[0] = 1;
    j = j + k;
}