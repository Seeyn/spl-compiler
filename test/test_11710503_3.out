Error type 2 at Line 5: undefined function: nononono
Error type 5 at Line 9: unmatching type on both sides of assignment
Error type 7 at Line 9: binary operation on non-number variables
